components/kyma-environment-broker/internal/process/provisioning/event_hub_step.go

```diff
@@ -28,6 +28,9 @@ const (
 	componentNameKnativeEventing      = "knative-eventing"
 	componentNameKnativeEventingKafka = "knative-eventing-kafka"
 	kafkaProvider                     = "azure"
+
+	// prefix is added before the created Azure resources
+	prefix = "k"
```

Comment:
Looks good to me. Can we have a comment or link to [Azure](https://docs.microsoft.com/en-us/rest/api/servicebus/create-namespace) to describe why we are prefixing?